src/Control/Monad/Eff/JQuery.purs

```diff
@@ -63,6 +118,24 @@ foreign import setAttr
   -> JQuery
   -> Eff (dom :: DOM | eff) Unit
 
+foreign import getAttrImpl
+  :: forall eff
+   . String
+  -> JQuery
+  -> Eff (dom :: DOM | eff) Foreign
+
+-- | Get an attribute value.
+getAttr 
+  :: forall eff
+   . String
+   -> JQuery
+   -> Eff (dom :: DOM | eff) (Maybe String)
+getAttr str jq = map foreignToString (getAttrImpl str jq)
+  where foreignToString f =
+          if isUndefined f
+            then Nothing
+            else Just (unsafeFromForeign f)
```

Comment:
Yes, meanwhile the attributeName is string, it always return string|undefined
I tested with `$("").attr("")`, `$("a").attr("href")` the only case that fails is `$("#").attr("")`  but the exception is in `$("#")`.
And copycat of: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jquery/index.d.ts#L271